src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/factory/StandardPlayerRequestResolverFactory.java

```diff
@@ -51,11 +51,11 @@ public StandardPlayerRequestResolver getNewInstance(
                                                  @NotNull final Object... context)
       throws IllegalArgumentException
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.IllegalArgumentException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)